src/components/llama-header.ts

```diff
@@ -15,14 +15,29 @@
  */
 
 import { LitElement, html } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { customElement, property } from 'lit/decorators.js';
 
 @customElement('llama-header')
 export class LlamaHeader extends LitElement {
+  @property({ type: String }) title: string;
+
+  constructor() {
```

Comment:
I don't think this constructor is necessary - you can assign a default value to the `title` property when it's declared

Reply:
Fair, I found warnings against this when I implemented it, but I guess it's OK now.. https://lit.dev/docs/v1/components/properties/#initialize-property-values-when-using-decorators